profile_library/ikea/ORMANAS LED Strip/model.json

```diff
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-11-09T00:06:13.324130",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "FGWP102, Fibargroup",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.15.2:docker"
+  },
+  "name": "ORMANAS LED Strip",
+  "standby_power": 0
```

Comment:
```suggestion
  "standby_power": 0.25
```

Standby power can never be 0, probably your smartplug was unable to pick it up because consumption is below the threshold which the plug is able to measure.
When you were unable to measure I suggest to set it to 0.25 which will be a good estimation.

Reply:
right, that is an omission, sorry.

checking the strip once more, I can see the meter returning a toggling state between 0 and 0.1
so I am not sure the 0.25 would not be too much.

![Nov-22-2024 11-20-29](https://github.com/user-attachments/assets/5b4840b8-eb8c-48e1-8931-81e805a9201a)

setting it to red on 1% increases to

<img width="143" alt="Scherm­afbeelding 2024-11-22 om 11 22 12" src="https://github.com/user-attachments/assets/3e520f62-d2a8-456a-a381-a11c58c622e9">


so yes, the meter does have that sensitivity.

ill leave it up ti you to decide whether I should set 0.1 a standby power reading, or make it higher as a guess.

Reply:
btw, there is an additional aspect of the metering device. the hs measurements where done using a Greenwave Single Socket, while the color temps where done with the mentioned Fibaro plug. Should we add that info in the manifest, and if yes, how?

Reply:
> btw, there is an additional aspect of the metering device. the hs measurements where done using a Greenwave Single Socket, while the color temps where done with the mentioned Fibaro plug. Should we add that info in the manifest, and if yes, how?

`measure_device` is a free text field, so you can add both devices there.

I have set standby power to 0.1 now.

When you can do the changes to measure device in the JSON the MR is complete and we can merge.